app/Http/Controllers/LanguageController.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class LanguageController extends Controller
+{
+    public function language($language){
+        session()->put('language', $language);
+       return redirect()->back();
```

Comment:
```suggestion
        session()->put('language', $language);
        
        return redirect()->back();
```
1 dòng trống trước return